### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by katieannemills** [isop-phmx/studyGroup#1]
Hi there - thanks for forking the Study Group repo! Let me know if you have any questions on how to get started - happy to help. First things first - now that you've got the website all set up, things go smoothest when you've got a team of people helping organize events and reach out to your community. Find some co-organizers and you'll be off to a great start!

For (tons) more information, don't miss the [Study Group Handbook](http://mozillascience.github.io/studyGroupHandbook/index.html), which includes all the collected wisdom on organizing a Study Group (and a handy link to lesson plans in the appendix).

Finally, jump on the [organzier's mailing list](https://mail.mozilla.org/listinfo/msl-study-groups) - this list is for Study Group leaders worldwide to interact with each other, and we use it to send invitations to our monthly virtual meeting to share back questions and ideas.

Let me know when you've got your first meeting booked, and I'll add you to the global [Map of Study Groups](https://github.com/mozillascience/studyGroupLessons/blob/master/whereWeAre.geojson) - thanks again, let me know!

**Comment by vjd** [comment]
@BillMills Thanks for the message! This is a great platform you provided and I am looking forward to hack into different aspects of this to work for our community - Pharmacometrics. Just to let you know that I will be tweaking the studyGroupHandbook and studyGroupLessons repos after forking them to meet our needs. Will keep you posted of our progress.

**Comment by katieannemills** [comment]
Sounds good! All our repos rely on ideas from people like you to get even better, so fork away and let us know what works for you! Also, don't miss our monthly online worldwide hangout for Study Group leaders to talk about their ideas (next one in just a bit over 12 hours!). Check out the details here and hopefully we see you there: https://public.etherpad-mozilla.org/p/mozilla-studygroup-leads-january-2016

**Comment by vjd** [comment]
@BillMills I've been working on getting things moving at our end and testing out the website. I am not able to figure out why the entries in the `_posts/` regarding events that I created in the issues are not showing up on the main page. Any tips?

**Comment by katieannemills** [comment]
yep, this is due to GitHub switching to jekyll 3.0 - we need to add

```
future: true
```

to the bottom of `_config.yml` (as I think you found on twitter earlier :) I'm going to merge this change upstream - thanks for pointing it out!
